### PR TITLE
Feature: multibranch pipeline support

### PR DESCRIPTION
**Pull request by vetal2409**

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/environmentdashboard/DeploymentView.java [hunk 1]
```diff
@@ -2,60 +2,85 @@
...
+        if (item instanceof WorkflowMultiBranchProject) {
+            runs = ((WorkflowMultiBranchProject) item)
+                    .getItems()
+                    .stream()
+                    .map(Job::getBuilds)
+                    .flatMap(Collection::stream)
+                    .collect(Collectors.toList());
+        } else if (item instanceof WorkflowJob) {
+            runs = ((WorkflowJob) item).getBuilds();
         }
```
Comment (StyleT) [inline src/main/java/org/jenkinsci/plugins/environmentdashboard/DeploymentView.java:47]: Shouldn't we return empty list for unexpected types of jobs?

**Review by StyleT — CHANGES_REQUESTED** [review]
Could you also pls remove TODO section from README?

File: src/main/java/org/jenkinsci/plugins/environmentdashboard/DeploymentView.java (hunk 1 above)
Comment (vetal2409) [reply src/main/java/org/jenkinsci/plugins/environmentdashboard/DeploymentView.java:47]: We do that. The first line initializes an empty List<WorkflowRun> and then we convert it to the empty List<Environment>. Then we filtered jobs with empty environments at all in the getUnits() method

Comment (StyleT) [reply src/main/java/org/jenkinsci/plugins/environmentdashboard/DeploymentView.java:47]: you're right, sorry, missed that line...

**Review by StyleT — APPROVED** [review]

**Review by panpuchkov — APPROVED** [review]

**Pull request merged**